status-bar/android/src/main/java/com/capacitorjs/plugins/statusbar/StatusBarPlugin.java

```diff
@@ -15,7 +16,35 @@ public class StatusBarPlugin extends Plugin {
 
     @Override
     public void load() {
-        implementation = new StatusBar(getActivity());
+        StatusBarConfig config = getStatusBarConfig();
+        implementation = new StatusBar(getActivity(), config);
+    }
+
+    private StatusBarConfig getStatusBarConfig() {
+        StatusBarConfig config = new StatusBarConfig();
+        String backgroundColor = getConfig().getString("StatusBarBackgroundColor");
```

Comment:
The name of the configuration string can probably just be `backgroundColor` (same with the others and on iOS).  It should be assumed that this is for the StatusBar plugin.